test/stream/api/sink/list-002-ok/009-list-limit-2-page-004/request.json

```diff
@@ -0,0 +1,7 @@
+{
+  "path": "/v1/branches/%%TEST_DEFAULT_BRANCH_ID%%/sources/my-source/sinks?limit=2&sinceId=my-sink-5",
```

Comment:
If I use `sinceId=my-sink-4` I sould receive `my-sink-4` in response. Am I right?

Reply:
You can rename the parameter in the URL to `afterId`, if you want.

Reply:
@jachym-tousek-keboola please create a task for the renaming